packages/fxa-payments-server/src/index.scss

```diff
@@ -41,13 +41,24 @@ h3.billing-title {
 }
 
 .main-content {
-  display: flex;
-  flex-direction: column-reverse;
-  padding-top: $header-height;
+  display: grid;
```

Comment:
Did you also want to change other flex items like `plan-details`? (I noticed with FPN, we probably need a gutter between the title and price)

I'm kind of indifferent on having grid/flex mixed right now because we haven't established a preferred pattern (but we will soon!) so I think flex + grid here is OK in the meantime, but _maybe_ it should wait? We'll either need to refactor this back to flexbox or finish refactoring the rest of payments to grid. What visual differences did this make for us, what issues does this PR fix with the layout?

![image](https://user-images.githubusercontent.com/13018240/79000495-568e1000-7b12-11ea-9c89-d6fe597a3e6f.png)

Reply:
so this PR does a few things but the biggest are:
1. It makes the payment success page and payment page behave the same at the same browser widths. previously the layout of the success page only showed up on mobile
2. changes the way the page looks on mobile views so that the details card is sized correctly
3. removes unnesc. border radii